pyproject.toml

```diff
@@ -58,6 +61,7 @@ lint = "pre-commit run --all-files"
 precommit = "pre-commit install"
 build = "docker build -t ghcr.io/python-discord/bot:latest -f Dockerfile ."
 push = "docker push ghcr.io/python-discord/bot:latest"
-test = "coverage run -m unittest"
+test-nocov = "pytest -n auto"
+test = "pytest -n auto --cov-report= --cov"
```

Comment:
What's the reason for suppressing the coverage printing?

Reply:
I wanted to maintain as much of the original interface as possible. The coverage also takes up a lot of space.

Reply:
Ah, I must've misremembered. Thought it was printed as a part of the test command before, but looks like it always was a separate step.